src/components/dialog/dialog.stories.tsx

```diff
@@ -45,8 +45,9 @@ export const DefaultStory: ComponentStory<typeof Dialog> = (args) => {
 
 DefaultStory.args = {
   leftAction: {
-    label: 'Remove',
-    level: 'destructive',
+    label: 'Toggle me',
```

Comment:
I wouldn't change this default story as it is quite rare and more usual to use a destructive button there.

Maybe just add another story?